Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "rust-spiffe"
+version = "0.1.0"
+authors = ["Max Lambrecht <maxlambrecht@gmail.com>"]
+edition = "2018"
+license = "Apache-2.0"
+categories = ["crytography"]
+keywords = ["SPIFFE", "X509", "JWT"]
+readme = "README.md"
+repository = "https://github.com/maxlambrecht/rust-spiffe"
+description = "Rust client library implementation for SPIFFE"
+
+[dependencies]
+grpcio = { version = "0.6", default-features = false, features = ["protobuf-codec"] }
```

Comment:
Do you want to bump that to at least `0.8` in order to include https://github.com/tikv/grpc-rs/pull/499? That would boost compilation time and also help us in Parsec 👍

Reply:
Sure! Bumped it to `0.9`. For some reason it needs `grpcio-sys = "0.9.0"` specified as `build-dependency`, otherwise the build throws an error.